test/test-reimbursementController.js

```diff
@@ -63,7 +63,6 @@ describe('reimbursementController', () => {
     });
 
     afterEach(async () => {
-      await clean.cleanAllPsychologists();
```

Comment:
* Why not clean psy anymore?
* Below there are 2 calls like this : `await dbPsychologists.savePsychologistInPG([clean.getOnePsy(...)]);` we may replace them by `clean.insertOnePsy(...)`

Reply:
because they are automatically cleaned by the university